.gitignore

```diff
@@ -56,3 +56,8 @@ jmh-benchmarks/src/main/generated
 raft/.jqwik-database
 **/src/generated
 **/src/generated-test
+
+.vim
+.bloop/
+.metals/
+bin/
```

Comment:
I would remove these gitignore patterns from the project. Git allows you to have personal gitignore patterns.

Reply:
Yes. Pushed by mistake. Fixed in next version.